src/scripts.js

```diff
@@ -35,13 +35,16 @@ function generateRandomId() {
 
 function displayPastTrips() {
```

Comment:
This is wild! Good work! I used classes and methods for mine to dry it up a bit, but I'm very impressed by what all you accomplished in this one function.